editor/src/components/canvas/canvas-strategies/shared-absolute-resize-strategy-helpers.ts

```diff
@@ -79,3 +88,372 @@ function pinsForEdgePosition(edgePosition: EdgePosition): AbsolutePin[] {
 
   return [...horizontalPins, ...verticalPins]
 }
+
+type IsCenterBased = 'center-based' | 'non-center-based'
```

Comment:
Probably worth exporting this and using it in the function parameters.

Reply:
I'm not sure which function you mean. I use it in all new `resizeBoundingBox...` and `snapping` functions, and I don't know of other functions which would need this.

Reply:
I can see it here for example: https://github.com/concrete-utopia/utopia/pull/2186/files#diff-95ecc07766bb6e984683db3702bb6b6d359a383a34ff254bff8f672c24193924R201